hs_geographic_feature_resource/page_processors.py

```diff
@@ -103,6 +103,7 @@ def landing_page(request, page):
         context['ori_coverage_form'] = ori_coverage_form
         context['geom_information_form'] = geom_information_form
 
+    context['edit_mode'] = edit_resource
```

Comment:
I am not still sure why the front end needs to know view/edit mode. Why can't you display the popup when the user tries to delete the file. May be I am missing something. After all it is a small change. So +1.